src/actions/DeepLinkingActions.js

```diff
@@ -102,6 +102,23 @@ function handleLink(dispatch: Dispatch, state: RootState, link: DeepLink): boole
       return true
     }
 
+    case 'azteco': {
+      if (!hasCurrentWallet) return false
+      const edgeWallet = currencyWallets[selectedWalletId]
+      if (edgeWallet.currencyInfo.currencyCode !== 'BTC') return false // prompt to choose btc wallet?
+      const address = await edgeWallet.getReceiveAddress()
+      const response = fetch(`${link.uri}${address.publicAddress}`)
+      if (response.ok) {
+        showToast('Success!')
```

Comment:
Translate  strings

Reply:
Change to "You've redeemed an Azteco bitcoin card. Funds should arrive shortly."